onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -93,39 +67,121 @@ class LoggingWrapper : public ISink {
   void* logger_param_;
 };
 
+struct OrtEnv {
+ public:
+  struct LoggingManagerConstructionInfo {
+    LoggingManagerConstructionInfo(OrtLoggingFunction logging_function1,
+                                   void* logger_param1,
+                                   OrtLoggingLevel default_warning_level1,
+                                   const char* logid1)
+        : logging_function(logging_function1),
+          logger_param(logger_param1),
+          default_warning_level(default_warning_level1),
+          logid(logid1) {}
+    OrtLoggingFunction logging_function{};
+    void* logger_param{};
```

Comment:
Would it make sense to delete the default ctor and just initialize all the members in the ctor you have?

Reply:
Sure. I didn't have the constructor before, but centos didn't allow aggregate initialization of objects. Can do in another PR.